examples/vg-specs/area.vg.json

```diff
@@ -33,6 +33,9 @@
                     ],
                     "fields": [
                         "count"
+                    ],
+                    "as": [
```

Comment:
(It's way more maintainable because AggregateNode from encoding become way easier to merge with AggregateNode from transform)